app/templates/components/flash-messages/fixed-position.hbs

```diff
@@ -0,0 +1,11 @@
+{{#each flashMessages.queue as |flash|}}
```

Comment:
Not critical for this PR to be mergeable, but could we not use a filter here?

Reply:
You mean to filter-by flash.fixed in this case and reject-by in the other?

Reply:
Yes, that's what I had in mind.